packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/dashboard/manage-nsx-edges/dedicatedCloud-datacenter-manage-nsx.html

```diff
@@ -0,0 +1,104 @@
+<oui-header
+    heading="{{'dedicatedCloud_manage_nsx_edge_title' | translate}}"
+    description="{{ 'dedicatedCloud_tab_datacenter_global_information' | translate}}"
+>
+    <oui-back-button data-on-click="$ctrl.goBack()"> </oui-back-button>
```

Comment:
No cancel method (to handle tracking) ?

Reply:
A call to trackClick is made inside the goback function in resolve of route file :
```
goBack: /* @ngInject */ (trackClick, goBackToDashboard) => () => {
    trackClick('::cancel');
    return goBackToDashboard();
}
```